llvm/utils/TableGen/GlobalISelEmitter.cpp

```diff
@@ -1248,10 +1244,6 @@ Expected<action_iterator> GlobalISelEmitter::importExplicitUseRenderer(
       DstMIBuilder.addRenderer<CopyRenderer>(Dst.getName());
       return InsertPt;
     }
-    if (Dst.getOperator()->getName() == "tframeindex") {
```

Comment:
This should be the same as the time case above?

Reply:
I'm not sure. Technically, yes, it would match the behavior of SelectionDAG.
On the other hand, there are no strict equivalent of `frameindex` / `tframeindex` in GISel. There is just one `G_FRAME_INDEX`, which behaves more like `G_CONSTANT` in that it is a separate instruction rather than just a flavor of machine operand. The same is true for G_GLOBAL_VALUE, G_CONSTANT_POOL, maybe some others.

Adding CopyRenderer is the default behavior that should be taken if a node doesn't require a special treatment. That is, the check above and the removed check are just whitelisting supported leaf-like nodes. So, if we want to allow this node, the check should be kept. If we want to make it clear that the node is unsupported, it should be removed. I have no strong preference.

Reply:
OK, considering there are other `t*` nodes, I'll just remove the check and make that and add `CopyRenderer` unconditionally. There is no good reason to treat `tframeindex` differently from other `t*` nodes, and whitelisting them isn't a scalable approach.

Reply:
Nah, this needs some larger refactoring. Will do it in a separate PR.
I'll restore the check for now.

Reply:
There aren't that many of these, and they are kind of a special case. It's only the constant-like opcodes have target equivalents